app/assets/javascripts/discourse/app/components/form-template-field/upload.gjs

```diff
@@ -0,0 +1,122 @@
+import Component from "@ember/component";
+import UppyUploadMixin from "discourse/mixins/uppy-upload";
+import { computed } from "@ember/object";
+import { htmlSafe } from "@ember/template";
+import { tracked } from "@glimmer/tracking";
+import { dasherize } from "@ember/string";
+import { isAudio, isImage, isVideo } from "discourse/lib/uploads";
+import PickFilesButton from "discourse/components/pick-files-button";
+import icon from "discourse-common/helpers/d-icon";
```

Comment:
```suggestion
import { tracked } from "@glimmer/tracking";
import Component from "@ember/component";
import { computed } from "@ember/object";
import { dasherize } from "@ember/string";
import { htmlSafe } from "@ember/template";
import PickFilesButton from "discourse/components/pick-files-button";
import { isAudio, isImage, isVideo } from "discourse/lib/uploads";
import UppyUploadMixin from "discourse/mixins/uppy-upload";
import icon from "discourse-common/helpers/d-icon";
```

I think you can use autofix to sort this correctly like so

Reply:
Oh yeah, I pushed this change right after I asked for your review, sorry about that!